src/nvim/tui/tui.c

```diff
@@ -1236,18 +1236,6 @@ static int unibi_find_ext_str(unibi_term *ut, const char *name)
   return -1;
 }
 
-static int unibi_find_ext_bool(unibi_term *ut, const char *name)
```

Comment:
If nothing's using it, it can be removed, but the justification in your commit was wrong. :)